src/utilities/add-remove.test.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const { expect, test } = require('@jest/globals');
+
+describe('Test add', () => {
+  // Arrange --------------------------------------------------------------------------->
+  document.body.innerHTML = '<div>'
+    + '<input type="text" name="task" id="task" class="input-task" value="Hello">'
+    + '<ul class="container-list"></ul>'
+    + '</div>';
+  const event = {
+    keyCode: 13,
+  };
+
+  class LocalStorage {
+    static list = [];
+
+    static setDataLocalStorage(item) {
+      this.list = item;
+    }
+
+    static getDataLocalStorage() {
+      return this.list;
+    }
+  }
+
+  class Task {
+    constructor(description, status, index) {
+      this.description = description;
+      this.status = status;
+      this.index = index;
+    }
```

Comment:
[OPTIONAL]
- I would recommend to use separate file to use mock for funtions that used `localStorage`, `DOM` or `external APIs`.